hub/core/chunk.py

```diff
@@ -47,6 +50,49 @@ def __init__(
 
         self._data: Union[memoryview, bytearray] = data or bytearray()
 
+        # Decompressed caches
+        self._decompressed_samples: Optional[List[np.ndarray]] = None
+        self._decompressed_data: Optional[bytes] = None
+
+    def decompressed_samples(
```

Comment:
can we name these functions better? and elaborate further on what these functions do in the docstring?

Reply:
docstring added.